src/main/resources/db/migration/V1.3__warehouse_schema.sql

```diff
@@ -0,0 +1,17 @@
+alter table if exists warehous drop constraint if exists FK_warehous_address_id;
+drop sequence if exists warehous_sequence;
+drop table if exists warehous cascade;
+
+create sequence warehous_sequence start 1 increment 1;
+
+create table warehous (
```

Comment:
You have a typo in the word "warehous".
Rename please in all project warehous -> warehouse